src/main/java/org/simlar/simlarserver/services/createaccountservice/CreateAccountService.java

```diff
@@ -117,6 +117,17 @@ AccountRequest createAccountRequest(final String telephoneNumber, final String s
                 settingsService.getMaxRequestsTotalPerDay(),
                 "too many total create account requests within one day");
 
+        final List<CreateAccountSettingsService.Regional> regionals = settingsService.getRegionals();
+        if (!CollectionUtils.isEmpty(regionals)) {
+            for (final CreateAccountSettingsService.Regional regional: regionals) {
+                final String regionCode = regional.getRegionCode();
+                if (StringUtils.isNotEmpty(regionCode) && simlarId.get().startsWith('*' + regionCode)) {
```

Comment:
`'*' + regionCode` looks like a perfect candidate for a local variable.

Reply:
Thanks again and there is a new commit fixing this.